LabExT/View/Movement/CoordinatePairingsWindow.py

```diff
@@ -161,6 +166,24 @@ def __setup__(self) -> None:
                 frame, self.mover.output_calibration).pack(
                 side=TOP, fill=X)
 
+        if self.experiment_manager:
+            shortcuts_frame = CustomFrame(self._main_frame)
+            shortcuts_frame.pack(side=TOP, fill=X, pady=5)
+
+            search_for_peak_button = Button(
+                shortcuts_frame,
+                text="Perform Search for Peak...",
+                command=self.experiment_manager.main_window.open_peak_searcher())
```

Comment:
I think you only need to pass the function, not the function-call's-return here. i.e. delete parentheses at end.